projects/spectator/src/lib/types.ts

```diff
@@ -24,7 +24,7 @@ export interface QueryOptions<R> {
   parentSelector?: Type<any> | string;
 }
 
-export type EventEmitterType<P> = P extends EventEmitter<infer T> ? T : never;
+export type OutputType<P> = P extends EventEmitter<infer T> ? T : P extends OutputRef<infer T> ? T : never;
```

Comment:
This is a breaking change because it requires a specific version of Angular. I don't mind to release a major version but you should include it in the peer dependency and the commit message

Reply:
Understood! I'll push this later today or tomorrow. Thank you for following up so quickly on this 😄

Reply:
Added - it's a shame this type got changed in a minor version, since I do believe semantically it's a breaking change in Angular.

Any conditional types that relied on `extends InputSignal` broke. I suppose the Angular team considered this as an internal type (or simply forgot about it/thought it wasn't worth the breaking change).